README.md

```diff
@@ -12,21 +12,27 @@ Pick a preferred read speed and select a song. You'll get a breakdown of BPM dur
 
 ## Setup
 
-You'll need a working node + NPM installation to build the frontend and a working Python + virtualenv installation to work with the API server / simfile parser.
+### Requirements
 
-To serve the API + frontend, run:
+- NodeJS
+- Yarn
+- Python 3.7.5
+  - [pyenv](https://github.com/pyenv/pyenv) is probably the easiest
+
+Set up a virtual env and install dependencies:
 
 ```
-virtualenv venv && source venv/bin/activate
-pip install -r requirements.txt -r dev_requirements.txt
-cd frontend/ && npm install && npm run build && cd -
-honcho start
+pyenv install (This'll install 3.7.5 if you don't have it already)
```

Comment:
```suggestion
# Assuming you use Homebrew. Otherwise install pyenv / pyenv-virtualenv using your preferred package manager / method
brew install pyenv pyenv-virtualenv
pyenv install (This'll install 3.7.5 if you don't have it already)
```

Reply:
I left the installation part out as `pyenv` is linked above in the requirements, so someone can just go there if they don't have it.

Reply:
Was trying to cover `pyenv-virtualenv` specifically since I ended up Googling install instructions after getting an error that it was missing.

Reply:
(not a big deal or blocker though)